Documentation/tls/tls-certificates.md

```diff
@@ -0,0 +1,49 @@
+# Transport Layer Security (TLS) Certificates
+
+Tectonic secures all communications between cluster components. At install time, certificates can be provided in the following ways, from most automated to most customized:
+
+1. Installer generates a new CA, intermediate CAs and all certificates (default)
+2. Installer uses your provided CA (or intermediate) to generate all certificates
+3. Certificates are generated out-of-band (guides below) and provided to the Installer
+
+Read about the [TLS topology][tls-topology] for more technical information about the intermediate CAs used between components.
+
+## Components and Certificate Sources
+
+| Type of Traffic | Source of Certificate |
+|:----------|:----------------------|
+| API communication (external and internal) | Generated (default) </br> [User-provided][tls-identity] |
+| API to etcd cluster | Generated (default) </br> [User-provided][tls-etcd] |
+| Kubelet to anything | Generated (default) </br> [User-provided][tls-kube] |
```

Comment:
This includes the certs that the apiserver will use to secure it's tls enable endpoint. And the cert that the kubelet will use to authenticate to the apiserver. One of the challenges here is that the kubelet cert if configured will mean that all kubelets will authenticate with this same cert. They are not unique to each node.

Reply:
question: if the kubelet cert is *not* configured, would this still be the case using the default certs provided by Tectonic?